scripts/hash-exec-copy.sh

```diff
@@ -1,13 +1,34 @@
 #!/usr/bin/env bash
+
 set -e
 
-[[ "$1" == https://raw.githubusercontent.com/makerdao/community/*/governance/votes/*.md ]] || { echo "Please provide the correct exec copy link to hash (e.g. url=https://raw.githubusercontent.com/makerdao/community/<commit>/governance/votes/<file name>.md)"; exit 1; }
+if [[ -z "$1" ]]; then
+echo "Error: Please provide a date in YYYY-MM-DD format as an argument."
+exit 1
+fi
+
+# Check input date for correct format
+if ! [[ $1 =~ ^[0-9]{4}-[0-9]{2}-[0-9]{2}$ ]] ; then
+    echo 'Invalid date format. Please use yyyy-mm-dd'
```

Comment:
```suggestion
    echo 'Invalid date format. Please use YYYY-MM-DD'
```

If you care abt this